src/libs/ajax/Resources.ts

```diff
@@ -0,0 +1,14 @@
+import _ from 'lodash/fp'
```

Comment:
I switched the extension to TypeScript so that AzureStorage wouldn't complain about the return type of `workspaceControlledResources`.

I'm not sure this is actually the best file to put the new method in… putting it `AzureStorage` seemed strange, and the name "Resource" seemed appealing. But it's obviously a very different type of Resource than the Sam resource. None of the other files looked particularly appealing.

Reply:
Done.

Reply:
If you're using something from js in a ts file, and it doesn't make sense to convert it at the time, you can just cast the return from js to `any`. 
It's not ideal, but it might make sense sometimes while the codebase is in a transitional state (personally, I prefer what you did here - just converting the file to ts, and cleaning up as we go).  

If this is logically distinct from the other functionality, might it make sense to put it in a new file?